packages/react-instantsearch/src/connectors/connectHierarchicalMenu.js

```diff
@@ -86,7 +86,7 @@ const sortBy = ['name:asc'];
  * @propType {string[]} [rootPath=null] - The already selected and hidden path.
  * @propType {boolean} [showParentLevel=true] - Flag to set if the parent level should be displayed.
  * @providedPropType {function} refine - a function to toggle a refinement
- * @providedPropType {function} createURL - a function to generate a URL for the corresponding state
+ * @providedPropType {function} createURL - a function to generate a URL for the corresponding searchState
```

Comment:
when mentionning the concept is natural language, I would use `search state` instead of `searchState`.